extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -40,3 +40,13 @@ amp-sticky-ad {
 .-amp-sticky-ad-layout > amp-ad {
   display: block;
 }
+
+.-amp-sticky-ad-close-button {
+  position: absolute;
+  height: 33px;
+  width: 33px;
+  background-color: black;
+  opacity: 0.5;
+  right: 0px;
```

Comment:
actually the close button is aligned to left by default, and to right with `direction=rtl`.
I put `direction=rtl` to `amp-sticky-ad` to make alignment right by default.